benchmark/sirun/shimmer-startup/meta.json

```
@@ -3,7 +3,7 @@
   "run": "node index.js",
   "run_with_affinity": "bash -c \"taskset -c $CPU_AFFINITY node index.js\"",
   "cachegrind": false,
-  "iterations": 10,
+  "iterations": 5,
```

Iterations should actually normally be much higher. Otherwise the variance is going to be high. I would normally go for something like 100 or above.

```suggestion
  "iterations": 20,
```

---

+1, I would set iterations closer to 20, as 5 is definitely not enough.

> I would normally go for something like 100 or above.
Same. But given how long execution takes, we have to be a bit more laxed (sadly).